### PR TITLE
ISSD-4149 [PRM] Table breaking on UAT

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		"@clayui/link": "*",
 		"@clayui/loading-indicator": "^3.60.0",
 		"@clayui/modal": "^3.80.0",
-		"@clayui/pagination-bar": "^3.70.0",
+		"@clayui/pagination-bar": "3.52.0",
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json:16]: It looks like the latest release for clay pagination-bar is [3.114](https://www.npmjs.com/package/@clayui/pagination-bar?activeTab=versions), have we tried that to see if it fixes it? also, if it is a bug we should be reporting to clay to fix (even if we temporarily need to go down a version)

Comment (thaynam) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json:16]: @ryanschuhler yes, Clarice reported the error a few months back, this is the [thread](https://liferay.slack.com/archives/C049XG98GQL/p1706028071611969). I am not sure if the fixed in newer versions, but we could try to deploy the current version in UAT and see if it works, what do you think?

Comment (ryanschuhler) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json:16]: sure, ideally we can replicate the issue locally and then see if updating the version fixes it, but if you want to try it on uat that could be fine as well. I am not sure if that thread ever became a ticket or was fixed, so if we see the issue still in the latest version we should create a ticket in jira for it

Comment (thaynam) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json:16]: @abfranca I will hold this change for now, and deploy the latest  clay pagination-bar [3.114](https://www.npmjs.com/package/@clayui/pagination-bar?activeTab=versions) in UAT to see if it fixes the issue. lets keep a close look at it. thnx!

Comment (thaynam) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/package.json:16]: @ryanschuhler we were not able to replicate it locally, it remains a problem only in LXC. Clarice had the same problem as well, hopefully the fixed in the laterst. we will keep you informed. thnx!

**Pull request closed without merging**